### DISCUSSION TITLE
Unexpected 404 when render page URL with / (slash) at the end

### DISCUSSION THREAD

**Discussion by theodore-zd** [vercel/next.js#13860] [Help] [answered]
# Bug report
**Unexpected 404 when rendering page URL with / (slash) at the end**

## Describe the bug
unexpected result when / added to end of page URL

## Expected
example (http://localhost:3000/home) 
renders "./pages/home.js" or "./pages/home/index.js"
_(or some kind or redirect)_

## Result
example (http://localhost:3000/home/) 
returns 404

## To Reproduce
1. 'basic Nextjs & express project'
2.  'run dev or build & start'

## Expected behavior

expected to render  "./pages/home.js" or "./pages/home/index.js"

## System information

- OS: Windows running locally using nodemon
- Browser Chrome
- Version of Next.js: 9.3.6
- Version of Node.js: 10.15.1

I'm not sure whether this is the expected result or not it wasn't for me when I notice, perhaps this is the only way Nextjs can handle routing. 

 I find this to be rather strange UX and I have never seen this kind of issue with any other site or web app.

**Comment by jamesmosier** [comment, marked as answer]
There is a PR open to fix this 😄 

https://github.com/vercel/next.js/pull/13333

**Reply by theodore-zd** [reply]
Oh! awesome thanks you letting me know!
